packages/notification-services-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/notification-services-controller",
-  "version": "6.0.0",
+  "version": "6.0.1",
```

Comment:
This is a patch release for a recent notification fix.
The changelog doesn't seem to contain other changes that would interfere with this being a patch release.